### ISSUE TITLE
Cannot load the files whose titles contains "&"

### ISSUE THREAD

**Issue by PJR91** [hanya/BookmarksMenu#6]
Hi,
BookmarksMenu is a very useful tool.
However, I cannot load the files whose titles contains "&".
I first get twice a message telling me that the file NNNN does not exist (as a matter of fact the message says in French that message NNN "n'existe pas"). Finally, a new message says (in French) that I cannot access this object since the user's rights are not sufficient.
I have Libre Office 5.1.3.2 (x64) and the last update of Windows 10.
I have the same problem with Libre Office 5.1.3.2 (x32)

**Comment by PJR91** [comment]
.... I have not this problem when there is no "&" in the title ...

**Comment by hanya** [comment]
Do you mean you tried to open the file by Something - File function? Not a document which the office can open?
It seems "&|()" characters should be escaped by "^" or quoted by double quotation mark. 
Currently, the extension uses the following command to open some file not related to the office on the Windows system: 

> rundll32.exe url.dll,FileProtocolHandler FILE_PATH

I do not have Windows system to test myself now.

**Comment by PJR91** [comment]
```
Thank you for your interest in my problem.

I have installed Bookmarksmenu on three computers (1 HP desktop,
  1 HP laptop, 1 Asus hybrid), with Windows 10 as the OS and I use
  it with LibreOffice 5.1.3.2.

With the 2 laptops, 


  I cannot open with BMM files such as "Entre bêtes sauvages
    & monstres froids" and I get the message BMM1. As you can
    see on the attached file, the name of the file ("Entre bêtes
    sauvages ") stops just before the "&" on this message. If I
    click on OK, I get the same message a second time and, then, the
    message BMM2. 

  I can open other files such as "Préparation du café en grains
    avec cafetière à piston" or "Médiathèque".

With the desktop, I cannot open any file, even those without any
  French accentuation such as "Codes SFR 08-2014".
Cordially.
Le 30/05/2016 à 18:52, hanya a écrit :


  Do you mean you tried to open the file by Something - File
    function? Not a document which the office can open?
    It seems "&|()" characters should be escaped by "^" or
    quoted by double quotation mark. 
    Currently, the extension uses the following command to open some
    file not related to the office on the Windows system: 

    rundll32.exe url.dll,FileProtocolHandler FILE_PATH

  I do not have Windows system to test myself now.
  —
    You are receiving this because you authored the thread.
    Reply to this email directly, view
      it on GitHub, or mute
      the thread.
```

**Comment by hanya** [comment]
Do you mean you can not open ODF files having & character in your  file name? Or other kind of files?

**Comment by PJR91** [comment]
```
It depends on the computer:

  on 2 computers, I can not open ODF files having &
    character in their file names, but I can open other files (I did
    not other exotic characters),
  on 1 computer, I cannot open any file.

BTW, this did not change after your last update of the extension
  (1.0.13).


Le 02/06/2016 à 19:12, hanya a écrit :


  Do you mean you can not open ODF files having & character
    in your file name? Or other kind of files?
  —
    You are receiving this because you authored the thread.
    Reply to this email directly, view
      it on GitHub, or mute
      the thread.
```

**Comment by hanya** [comment]
Fixed by #9
Use version 1.0.15.
